packages/account/src/Components/api-token/api-token-table.tsx

```diff
@@ -104,14 +107,15 @@ const ApiTokenTable = () => {
         );
     }
 
+    const api_token_table_header_title = ['Name', 'Token', 'Scopes', 'Last used'];
+
     return (
         <table className='da-api-token__table'>
             <thead>
                 <tr className='da-api-token__table-header-row'>
-                    <ApiTokenTableRowHeader text={<Localize i18n_default_text='Name' />} />
-                    <ApiTokenTableRowHeader text={<Localize i18n_default_text='Token' />} />
-                    <ApiTokenTableRowHeader text={<Localize i18n_default_text='Scopes' />} />
-                    <ApiTokenTableRowHeader text={<Localize i18n_default_text='Last used' />} />
+                    {api_token_table_header_title.map(title => (
+                        <ApiTokenTableRowHeader key={title} text={<Localize i18n_default_text={title} />} />
```

Comment:
Does using `<Localize/> `this way? Please re-verify. `<Localize i18n_default_text={'{{text}}'} values={{title} />`. I think dynamic value need to be passed to values prop of Localize component

Reply:
It's working without passing it as values also.

Reply:
This is not the effective way to use this as on build time, anything passed to `i18n_default_text` is extracted by a script and if any string which is not translated yet will be submitted as is i.e. if `Name` isn't translated yet, translation team will get `title` to translate.

And in our case, the extraction will skip this as it's not starting from single or double quotes

Reply:
So, I suggest you can create a function to return the title as `Localize` component by passing the name

something like this:
```suggestion
const api_token_table_header_title = ['name', 'token', 'scopes', 'last_used'];


const getTitle = (key) => {
  const titles = {
  'name': <Localize i18n_default_text="Name"/ >,
   ...
  }
  return titles[key]
}
...
{api_token_table_header_title.map(title => (
     <ApiTokenTableRowHeader key={title} text={<Localize i18n_default_text={getTitle(title)} />} />
))}

Reply:
But won't this return <Localize i18n_default_text="Name"/ > inside ext={<Localize i18n_default_text={getTitle(title)} />}. The array already has strings inside so we basically passing them..and the stings are already translated so it works.

Reply:
My bad 
I meant like this actually

```
{api_token_table_header_title.map(title => (
	     <ApiTokenTableRowHeader key={title} text={getTitle(title)} />
))}